src/app-layout/index.tsx

```diff
@@ -197,6 +197,14 @@ const OldAppLayout = React.forwardRef(
       }
     };
 
+    useEffect(() => {
+      // To guarantee an optimal UX on mobile, we forcely close the navigation on this viewport
```

Comment:
(minor) "guarantee" and "optimal" are some pretty big words, and don't really explain the motivation. I would say something like "Close navigation drawer on mobile so that the main content is visible"

Reply:
Good point. Not really trying to sell a feature here. I'll update